lib/components/form/SiteDetails.tsx

```diff
@@ -33,7 +30,7 @@ const SiteDetails: FC<SiteDetailsProps> = ({ site }) => {
 
   const nextPageCallback = () => {
     if (page === Page.Details) {
-      router.push(mobile ? '/sites' : '/dashboard');
+      router.push('/dashboard');
```

Comment:
Is there a reason for removing this? I added it since desktop shouldn't use the sites page.

Reply:
yeah no i just got confused gonna put it back